misc/config_tools/xforms/vm_configurations.h.xsl

```diff
@@ -37,12 +37,12 @@
   </xsl:template>
 
   <xsl:template name ="vm_count">
-    <xsl:value-of select="acrn:comment('SERVICE_VM_NUM can only be 0U or 1U; When SERVICE_VM_NUM is 0U, MAX_POST_VM_NUM must be 0U too; MAX_POST_VM_NUM must be bigger than CONFIG_MAX_KATA_VM_NUM.')" />
+    <xsl:value-of select="acrn:comment('SERVICE_VM_NUM can only be 0U or 1U; When SERVICE_VM_NUM is 0U, MAX_POST_VM_NUM must be 0U too.')" />
```

Comment:
Could this be done by the tool automatically, i.e. set `MAX_POST_VM_NUM` to 0

Reply:
```suggestion
    <xsl:value-of select="acrn:comment('SERVICE_VM_NUM can only be 0 or 1; When SERVICE_VM_NUM is 0, MAX_POST_VM_NUM must be 0 too.')" />
```

Reply:
will update

Reply:
> Could this be done by the tool automatically, i.e. set `MAX_POST_VM_NUM` to 0

We think it will be better to have assertion than hide such logic in configtool from a maintenance view.